docs/advanced/pool.rst

```diff
@@ -6,13 +6,13 @@ Connection pools
 ================
 
 A `connection pool`__ is an object managing a set of connections and allowing
-their use to functions needing one. Because the time to establish a new
+their use in functions needing one. Because the time to establish a new
 connection can be relatively long, keeping connections open can reduce the
-latency of a program operations.
+latency of program operations.
```

Comment:
maybe "the program" is better?

Reply:
Or even shorter: "keeping connections open can reduce latency."?

Reply:
Yes, why not :)

Reply:
+1 for the shorter version. Occam's Razor!